src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -836,39 +1180,46 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
     );
   };
 
-  public freezeMinting = async (params: TxParams) => {
+  public freezeIssuance = async (params: FreezeIssuanceParams) => {
     assert.assert(
-      await (await this.featureRegistryContract()).getFeatureStatus.callAsync(Features.FreezeMintingAllowed),
+      await (await this.featureRegistryContract()).getFeatureStatus.callAsync(Feature.FreezeMintingAllowed),
```

Comment:
This assertion is no longer being enforced on the contract side

Reply:
Indeed. Will push removal to branch